docs/06-concepts/02-models.md

```diff
@@ -243,18 +269,18 @@ fields:
   dateTimeDefault: DateTime, default=now
 
   ### Sets the default value for a boolean field.
-  boolDefault: bool, defaultModel=false, defaultDatabase=true
+  boolDefault: bool, defaultModel=false, defaultPersist=true
 
   ### Sets the default value for an integer field.
-  intDefault: int, defaultDatabase=20
+  intDefault: int, defaultPersist=20
 
   ### Sets the default value for a double field.
-  doubleDefault: double, default=10.5, defaultDatabase=20.5
+  doubleDefault: double, default=10.5, defaultPersist=20.5
 
   ### Sets the default value for a string field.
   stringDefault: String, default="This is a string", defaultModel="This is a string"
 ```
 
 ### Coming Soon
 
-- **UuidValue**: Support for default values for `UuidValue` field type is coming soon.
+- **Duration**: Support for default values for the `Duration` field type is coming soon.
```

Comment:
Suggestion: Remove any coming soon docs. I think it is generally a bad idea to add "coming soon" things in our docs. If we for some reason do not implement a feature it looks pretty bad with a bunch of coming soon things spread around that are not being implemented. 

Instead only describe functional features that can be used.